posthog/queries/trends/total_volume.py

```diff
@@ -50,7 +52,13 @@ def _total_volume_query(self, entity: Entity, filter: Filter, team: Team) -> Tup
         params = {**params, **math_params, **event_query_params}
 
         if filter.display in NON_TIME_SERIES_DISPLAY_TYPES:
-            content_sql = VOLUME_TOTAL_AGGREGATE_SQL.format(event_query=event_query, **content_sql_params)
+
+            if entity.math_property == "$session_duration":
```

Comment:
This feels a bit fragile to me. Not blocking, but I wonder if we should add a `math_property_type` that's specified as `session` in this case. We definitely want this if we generalize this to other property types - I guess it's just a matter of when we add it.

Reply:
Indeed. I did have that earlier, but removed it for now. We can bring it back whenever we add a second property